Cargo.toml

```diff
@@ -26,4 +26,5 @@ neon = ["blake3/neon"]
 blake3 = { version = "1.5.5", features = ["mmap", "rayon"] }
 hex = "0.4.3"
 pyo3 = { version = "0.25.0", features = ["extension-module"] }
-rayon = "1.10.0"
+rayon = "~1.10.0"
+rayon-core = "~1.12.1"
```

Comment:
@oconnor663 now that you bumped minimum rust toolchain ref https://github.com/oconnor663/blake3-py/commit/8b3e6bcc4a8dea3dcf09233db620eb950b96bdcf, i think this can be reverted

Reply:
`rust-version` was explicit in `Cargo.toml` ref #56

Reply:
not sure if that's still relevant now that we go above 1.78 anyway